### DISCUSSION TITLE
Video-downloadhelper Has Stopped Downloading Via YouTube (At Least For Me It Has)?

### DISCUSSION THREAD

**Discussion by keyonew** [aclap-dev/video-downloadhelper#1568] [Help and Q&A]
### Your question

Windows 11, Firefox 133.0.3, Video Downloadhelper Version 9.1.0.44 (stable), target mozilla, locale en-US.
Licensed copy.

The usual "Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error (it's anonymous!)." error on every video. EVERY video.

Have reloaded Firefox, tinkered with settings, reloaded VDH to no avail.
Video-downloadhelper appears to be broken (at least for me).

Anyone else experiencing issues?

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"<removed>"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:133.0) Gecko/20100101 Firefox/133.0
{
    "valid_license_message_has_been_displayed": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 5
}

**Comment by GrampaWildWilly** [comment]
Go to the home page of this forum:

https://github.com/aclap-dev/video-downloadhelper/discussions?discussions_q=

At the top there is an announcement with this title:

Video not detected on YouTube

Click that.  Read what you find.

**Reply by keyonew** [reply]
Hi,
Thanks for the comment.
Yes, I read through the thread in detail. Tried almost all of the VPN country options, turning my VPN off, logging in/out of YT. Still not working.

**Reply by Mattmjs** [reply]
Check your anti virus and firewall aren't blocking the companion app. Try restarting 
the computer.

**Comment by 5EJKER** [comment]
hey I have a question?

do you still have this version installed on firefox

video download helper

[9.1.0.44](http://9.1.0.44/)

because now the creator has removed the versions that worked for free and left only the old 7.0

9.1.0.44-generally there is probably no better plugin for downloading movies. it finds everything it can practically always

but recently when I transferred the profile from browser to browser

it unfortunately

something probably overwrote it

and it updated

if you have this version

could you send me the file for it. it should be called like that. "(b9db16a4-6edc-47ec-a1f4-b86292ed211d).xpi" 

and it is in the %appdata% location and there in firefox in profiles --> extension 

I also found for chrome someone saved this version so I can send it to you. but for firefox it is nowhere. 

https://www.transfernow.net/dl/Videodowloaderdlachrome

**Reply by Mattmjs** [reply]
I see that most old versions are gone, only 5 older versions from version 7 and up then it's the current 9.3.7.2
Either Mozilla has done this or the developer but for what reason I don't know. Try the latest version and see how it goes.

**Reply by jcfrog** [reply]
This was a request from Mozilla, as these versions did not comply with the new policies. This is why the latest version has been delayed several times in order to comply with the new policies. Not sure the old ones will be republished.

**Comment by TeeMars** [comment]
So is there any simple workaround for downloading YT videos?

**Reply by jcfrog** [reply]
Latest Edge or Firefox extensions should work with Youtube.
